code/modules/assembly/anomalies.dm

```diff
@@ -117,25 +123,42 @@
 /obj/item/assembly/signaler/anomaly/phantom
 	name = "\improper phantom anomaly core"
 	desc = "The stabilized core of a phantom anomaly. It quietly screams."
+	grind_results = list(/datum/reagent/blood = 20)
 
 /obj/item/assembly/signaler/anomaly/phantom/anomaly_core_detonation()
 	playsound(src,'sound/hallucinations/far_noise.ogg', 100, 0, 50, TRUE, TRUE)
-	visible_message(span_warning("[src] screams as it disappates!"))
+	visible_message(span_warning("[src] screams as it fades, trying to lash out!"))
+	for(var/mob/living/carbon/handsy in range(5, src))
+		if(handsy.stat != DEAD)
+			var/grab_dir = turn(handsy.dir, pick(-90, 90, 180, 180))
+			var/turf/spawn_turf = get_ranged_target_turf(handsy, grab_dir, 8)
+			if(!spawn_turf)
+				return
+			new /obj/effect/temp_visual/dir_setting/curse/grasp_portal(spawn_turf, handsy.dir)
+			playsound(spawn_turf, 'sound/effects/curse2.ogg', 80, TRUE, -1)
+			var/obj/projectile/curse_hand/phantom/hand = new (spawn_turf)
+			hand.preparePixelProjectile(handsy, spawn_turf)
+			if(QDELETED(hand))
+				return
+			hand.fire()
 	..()
 
 ///Plasmasoul Anomaly
 /obj/item/assembly/signaler/anomaly/plasmasoul
 	name = "\improper plasmasoul anomaly core"
 	desc = "The stabilized core of a plasmasoul anomaly. The air around it hisses."
+	grind_results = list(/datum/reagent/toxin/plasma = 50)
 
 /obj/item/assembly/signaler/anomaly/plasmasoul/anomaly_core_detonation()
-	visible_message(span_warning("[src] rapidly delaminates, leaving behind trace vapors!"))
+	visible_message(span_warning("[src] rapidly delaminates into plasma!"))
+	atmos_spawn_air("plasma=500;TEMP=[T20C]")
 	..()
 
 ///Pulsar Anomaly
 /obj/item/assembly/signaler/anomaly/pulsar
 	name = "\improper pulsar anomaly core"
 	desc = "The stabilized core of a pulsar anomaly. Electromagnetic crackles come off it."
+	grind_results = list(/datum/reagent/iron = 25, /datum/reagent/uranium = 25)
```

Comment:
are you Sure you want to do this to people

Reply:
I could be convinced otherwise.

Reply:
I do not see it lasting long but it would be funny